pkg/tkctl/cmd/cmd.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2019. PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"flag"
+	"github.com/pingcap/tidb-operator/pkg/tkctl/cmd/ctop"
+	"github.com/pingcap/tidb-operator/pkg/tkctl/cmd/debug"
+	"github.com/pingcap/tidb-operator/pkg/tkctl/cmd/get"
+	"github.com/pingcap/tidb-operator/pkg/tkctl/cmd/info"
+	"github.com/pingcap/tidb-operator/pkg/tkctl/cmd/list"
+	"github.com/pingcap/tidb-operator/pkg/tkctl/cmd/pdctl"
+	"github.com/pingcap/tidb-operator/pkg/tkctl/cmd/use"
+	"github.com/pingcap/tidb-operator/pkg/tkctl/config"
+	"github.com/spf13/cobra"
+	"io"
```

Comment:
go fmt

Reply:
I've ran `go fmt` and these seems not being considered as format errors.